plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -80,13 +81,22 @@ func (s *SchedulerServer) Run(_ []string) error {
 	}
 
 	go func() {
+		mux := http.NewServeMux()
 		if s.EnableProfiling {
```

Comment:
I'm fine with enabling it by default, but I think it should be consistent among all component (i.e. we should also enable it in apiserver and controller). What do you think?